pyproject.toml

```diff
@@ -78,7 +78,7 @@ testpaths = "tests"
 norecursedirs = ".git testing_config"
 
 [tool.codespell]
-skip = "Contributors.md"
+skip = "Contributors.md,tests/test_tuya.py"
```

Comment:
Can you remove this and use "inline ignores" now that https://github.com/zigpy/zha-device-handlers/pull/3257 is merged?
You'll need to rebase on `dev` (or merge `dev` into your branch) to pick up the codespell change from that PR.

Reply:
> Can you remove this and use "inline ignores" now that #3257 is merged? You'll need to rebase on `dev` (or merge `dev` into your branch) to pick up the codespell change from that PR.

It doesn't work, codespell version is still 2.2.6 in [yaml config file](https://github.com/zigpy/zha-device-handlers/blob/156aaa6bb704a13fe9500b833c293405fc5ffd79/.pre-commit-config.yaml#L3).